pkg/commands/copy.go

```diff
@@ -39,6 +40,11 @@ func (c *CopyCommand) ExecuteCommand(config *manifest.Schema2Config) error {
 	logrus.Infof("cmd: copy %s", srcs)
 	logrus.Infof("dest: %s", dest)
 
+	// Resolve from
+	if c.cmd.From != "" {
```

Comment:
Sorry, what do you mean by randomness?

Right now, if "from" is specified as a previous stage, we change the build context to be the path to the directory where files from that stage are saved. That directory should remain unchanged even if multiple copy commands refer back to it

Reply:
I was worried about a case like this:

```
FROM base1
RUN something

FROM base1
RUN somethingelse

FROM base2
COPY --from=0
```

But I guess we can't have conflicts here, because the users have to refer to them by index (or alias)